keras_nlp/api/models/__init__.py

```diff
@@ -248,6 +248,13 @@
     RobertaTextClassifierPreprocessor as RobertaPreprocessor,
 )
 from keras_nlp.src.models.roberta.roberta_tokenizer import RobertaTokenizer
+from keras_nlp.src.models.segment_anything.sam_backbone import SAMBackbone
```

Comment:
directory name should match symbols name. If we are going with `SAM` everywhere, which seems fine, let's call the directory `sam`.